tcp/pcb.ml

```diff
@@ -18,6 +18,8 @@
 open Lwt
 open Printf
 
+let () = Random.self_init ()
```

Comment:
Just wondering, could it matter if this Random were a different module from the Random that is passed as a parameter to the Make functor? I guess it must have been parameterised for a reason.